schema/divisions/defs.yaml

```diff
@@ -89,36 +89,22 @@ description: Common schema definitions for divisions theme
           type: string
           minLength: 1
           pattern: ^(\S.*)?\S$    # Leading and trailing whitespace are not allowed.
-    perspective:
-      description:
-        A political perspective from which the division is viewed
+    perspectives:
+      description: Collection of perspectives of single feature.
```

Comment:
Consider changing to *Political perspectives from which division is viewed.* ...